### ISSUE TITLE
ArtistID8-Motivation2

### ISSUE THREAD

**Issue by edgartdata** [PHAROS-photoarchives/Zeri-Foundation#15]
ArtistID8-Motivation2 line 104 and ArtistID8-Motivation1 line 102 seem to be the same but the examples are different.

**Comment by marilenadaquino** [comment]
Yes! I mean, they are the same concept, but line 102 includes the motivation supporting the accepted attribution, while 104 includes the motivation supporting a discarded attribution.
